cluster/images/hyperkube/Dockerfile

```diff
@@ -78,3 +75,7 @@ RUN ln -s /hyperkube /apiserver \
  && ln -s /hyperkube /kubelet \
  && ln -s /hyperkube /proxy \
  && ln -s /hyperkube /scheduler
+
+# Copy the hyperkube binary
+COPY hyperkube /hyperkube
```

Comment:
I do not think this can come before the `ln -s` lines above that are referencing this file